.env-devel

```diff
@@ -91,7 +91,7 @@ DIRECTOR_V2_DYNAMIC_SCHEDULER_CLOSE_SERVICES_VIA_FRONTEND_WHEN_CREDITS_LIMIT_REA
 DIRECTOR_V2_DYNAMIC_SIDECAR_SLEEP_AFTER_CONTAINER_REMOVAL=0
 DIRECTOR_V2_GENERIC_RESOURCE_PLACEMENT_CONSTRAINTS_SUBSTITUTIONS='{}'
 DIRECTOR_V2_HOST=director-v2
-DIRECTOR_V2_LOGLEVEL=WARNING
+DIRECTOR_V2_LOGLEVEL=INFO
```

Comment:
Minor:
why the changes in this file at all?
is this used in OPS? I thought it was not anymore?

some local computers might not like having suddenly many more logs. just a wonder.

Reply:
Which computers? I find it more useful when I start `make up-prod` when I see INFO logs.